argus/config/__init__.py

```diff
@@ -25,8 +25,8 @@
     option_class(CONFIG).register()
 
 _DEFAULT_CONFIG_FILES = [
-    config_file for config_file in ("argus.conf", "etc/argus/argus.conf",
-                                    "/etc/argus/argus.conf")
+    config_file for config_file in ("/etc/argus/argus.conf",
```

Comment:
I think you could let a note/comment above the code block, saying why we want `argus.conf` to have the highest priority.